subsys/bluetooth/host/conn_internal.h

```diff
@@ -160,6 +169,9 @@ struct bt_conn {
 #if defined(CONFIG_BT_BREDR)
 		struct bt_conn_br	br;
 		struct bt_conn_sco	sco;
+#endif
+#if defined(CONFIG_BT_AUDIO)
+		struct bt_conn_iso	iso;
```

Comment:
I'm not allowed to add a comment on a unchanged line: It seems that `bt_conn` `channels` field is also used for `bt_iso_chan` and not just L2CAP channels, so perhaps the comment on line 160 should be updated?

Reply:
Yep, just fixed that.